lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,630 @@
+        state=dict(type='str', default="present", choices=["absent", "present"]),
+        data_type=dict(type='str', choices=['bigint', 'integer', 'smallint']),
+        increment=dict(type='int'),
+        minvalue=dict(type='int', aliases=['min']),
+        maxvalue=dict(type='int', aliases=['max']),
+        start=dict(type='int'),
+        cache=dict(type='int'),
+        cycle=dict(type='bool'),
+        schema=dict(type='str'),
+        cascade=dict(type='bool'),
+        rename_to=dict(type='str'),
+        owner=dict(type='str'),
+        newschema=dict(type='str'),
+        db=dict(type='str', default='', aliases=['login_db', 'database']),
+        session_role=dict(type='str'),
+    )
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        mutually_exclusive=(('positional_args', 'named_args'),),
+        supports_check_mode=True,
+    )
+
+    sequence = module.params["sequence"]
+    state = module.params["state"]
+    data_type = module.params["data_type"]
+    increment = module.params["increment"]
+    minvalue = module.params["minvalue"]
+    maxvalue = module.params["maxvalue"]
+    start = module.params["start"]
+    cache = module.params["cache"]
+    cycle = module.params["cycle"]
+    cascade = module.params["cascade"]
+    rename_to = module.params["rename_to"]
+    schema = module.params["schema"]
+    owner = module.params["owner"]
+    newschema = module.params["newschema"]
+
+    # Check mutual exclusive parameters:
+    if state == 'absent' and (data_type or increment or minvalue or maxvalue or
+                              start or cache or cycle or
+                              rename_to or newschema or owner):
+        module.fail_json(msg="'%s': state=absent is mutually exclusive with: "
+                             "data_type, increment, minvalue, maxvalue, "
+                             "start, cache, cycle, "
+                             "rename_to, newschema or owner" % sequence)
+
+    if rename_to and (data_type or increment or minvalue or maxvalue or start
+                      or cache or cycle or cascade or newschema or owner):
+        module.fail_json(msg="'%s': rename_to is mutually exclusive with: "
+                             "data_type, increment, minvalue, maxvalue, "
+                             "start, cache, cycle, cascade, "
+                             "newschema or owner" % sequence)
+
+    if cascade and (data_type or increment or minvalue or maxvalue or start or
+                    cache or cycle or rename_to or
+                    newschema or owner):
+        module.fail_json(msg="'%s': cascade is mutually exclusive with: "
+                             "data_type, increment, minvalue, "
+                             "maxvalue, start, cache, cycle, "
+                             "rename_to, newschema "
+                             "or owner" % sequence)
+
+    db_connection = connect_to_db(module, autocommit=True)
```

Comment:
```suggestion
    db_connection = connect_to_db(module, autocommit=False)
```
Then you can remove # Change autocommit to False if check_mode and PSYCOPG2_VERSION constant from the top of the module